### PR TITLE
Change build to esm to support plaiceholder plugin

### PR DESCRIPTION
**Pull request by youngkidwarrior**
@0xBigBoss  Not sure how you feel about changing to esm modules for our next config. 

The library I'm using for progressive image loading is esm only. If you are against it, I can find another library that supports commonJS

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/114?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #113** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/113?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #114** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/114?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #115** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/115?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/114?utm_source=stack-comment).

**Review by 0xBigBoss — COMMENTED** [review]
it's fine, but we need to fix CI

File: apps/next/next.config.js [hunk 1]
```diff
@@ -11,7 +12,7 @@ const disableExtraction =
   boolVals[process.env.DISABLE_EXTRACTION] ?? process.env.NODE_ENV === 'development'
 
 const plugins = [
```
Comment (0xBigBoss) [inline apps/next/next.config.js:14]: add it to the plugins list here

**Review by 0xBigBoss — CHANGES_REQUESTED** [review]

File: apps/next/next.config.js (hunk 1 above)
Comment (youngkidwarrior) [reply apps/next/next.config.js:14]: You actually wrap the returned config in a higher order function `withPlaiceholder`

https://github.com/0xsend/sendapp/pull/114/files#diff-0f913df15d01006c7733a9e4969f6335577a8a1ca3e1669c0013f57361c9ba7bR98

https://plaiceholder.co/docs/plugins/next

Comment (0xBigBoss) [reply apps/next/next.config.js:14]: yeah it's exactly what the tamagui plugin does.

https://github.com/joe-bell/plaiceholder/blob/main/packages/next/src/index.ts

https://github.com/tamagui/tamagui/blob/master/packages/next-plugin/src/withTamagui.ts#L23

File: packages/playwright/tests/send.onboarded.spec.ts [hunk 2]
```diff
@@ -22,24 +22,13 @@ test('can send ETH to user on profile', async ({ page, seed }) => {
...
+  const profilePage = new ProfilePage(page, {
+    name: profile.name,
+    about: profile.about,
+  })
   await profilePage.visit(tag.name, expect)
   await expect(profilePage.sendButton).toBeVisible()
   await profilePage.sendButton.click()
-
-  // @todo create send form fixture
-  const sendDialog = page.getByRole('dialog', { name: 'Send' })
```
Comment (0xBigBoss) [inline packages/playwright/tests/send.onboarded.spec.ts:31]: What happened to this test?

File: packages/playwright/tests/send.onboarded.spec.ts [hunk 3]
```diff
@@ -50,25 +39,11 @@ test('can send USDC to user on profile', async ({ page, seed }) => {
...
```
Comment (0xBigBoss) [inline packages/playwright/tests/send.onboarded.spec.ts:59]: Here too?

Comment (youngkidwarrior) [reply packages/playwright/tests/send.onboarded.spec.ts:59]: I removed them because I thought this page didn't exist anymore

Comment (youngkidwarrior) [reply packages/playwright/tests/send.onboarded.spec.ts:59]: cc @0xBigBoss

Comment (youngkidwarrior) [reply packages/playwright/tests/send.onboarded.spec.ts]: think my local machine was lying to me and this wasn't the correct PR to change this. Still my changes to onboarding in future PRs will deprecate this playwright code, no?

**Comment by youngkidwarrior** [comment]
@0xBigBoss SUCCESS

**Review by 0xBigBoss — APPROVED** [review]

**Pull request merged**